src/handlers/dns/DnsStreamHandler.h

```diff
@@ -142,8 +144,8 @@ class DnsMetricsBucket final : public visor::AbstractMetricsBucket
         , _dns_slowXactIn("dns", "qname", {"xact", "in", "top_slow"}, "Top QNAMES in transactions where host is the server and transaction speed is slower than p90")
         , _dns_slowXactOut("dns", "qname", {"xact", "out", "top_slow"}, "Top QNAMES in transactions where host is the client and transaction speed is slower than p90")
     {
-        set_event_rate_info("dns", {"rates", "total"}, "Rate of all DNS wire packets (combined ingress and egress) per second");
-        set_num_events_info("dns", {"wire_packets", "total"}, "Total DNS wire packets");
+        set_event_rate_info("dns", {"rates", "pps_events"}, "Rate of all DNS wire packets (combined ingress and egress) before filtering per second");
```

Comment:
@rboucher-ns1 what do you think of the description?

Reply:
Is the "(combined ingress and egress)" necessary? 
I think "Rate of all DNS wire packets before filtering per second" is clear and complete. Or is "ingress and egress" necessary?

Reply:
cool, changed it!